plugins/modules/powervm_lpar_instance.py

```diff
@@ -1512,6 +1535,9 @@ def install_aix_os(module, params):
 
         if location_code:
             hmc.installOSFromNIM(location_code, nim_ip, nim_gateway, vm_ip, nim_vlan_id, nim_vlan_priority, nim_subnetmask, vm_name, profile_name, system_name)
+        elif vm_mac:
+            hmc.installOSFromNIM(location_code, nim_ip, nim_gateway, vm_ip, None, None, nim_subnetmask, vm_name, profile_name, system_name, vm_mac)
+            hmc.checkconsolelog(module, vm_ip, hmc_host, hmc_user, password, system_name, vm_name)
```

Comment:
Is this method valid only when user passes vm_mac? If no please enable the same in other places too like when used pass the loc_code and doesn't pass the loc_code and in VIOS module too

Reply:
yes when user pass vm_mac, check_console will run